packages/fluentui/docs/src/prototypes/table/ResponsiveTable.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Table, TableCell, TableProps } from '@fluentui/react-northstar';
+import calculateBreakpoints, { Column, Breakpoint } from '../../utils/calculateBreakpoints';
+import * as _ from 'lodash';
+
+interface ResponsiveTableProps extends TableProps {
+  label: string;
+  id?: string;
+  header: {
+    items: Column[];
+  };
+}
+
+const ResponsiveTable: React.FC<ResponsiveTableProps> = props => {
+  const {
+    rows,
+    label,
+    header: { items },
+  } = props;
+
+  const responsiveTableID = props.id || _.uniqueId(`${Table.className}__responsive-wrapper`);
```

Comment:
this way, id will regenerate on each render, can you save it in state so that it remains constant once generated?

Reply:
Done